src/MonteCarlo.jl

```diff
@@ -1,19 +1,19 @@
 module MonteCarlo
 
 using Reexport
-@reexport using MonteCarloObservable
+@reexport using MonteCarloObservable, TimerOutputs
```

Comment:
Why reexport TimerOutputs? I'd prefer to not do this.

Reply:
I wanted `print_timer()`, `reset_timer!()` and `TimerOutputs.enable_debug_timings(module)` to be available. Guess we can also hide them...

Reply:
I think it's fine to have to load TimerOutputs manually for printing the timings. Perhaps a function `enable_benchmarks` that wraps `TimerOutputs.enable_debug_timings(MonteCarlo)` would be handy.